libethereum/State.h

```diff
@@ -136,8 +136,8 @@ struct Change
     {}
 
     /// Helper constructor especially for new code change log.
-    Change(Address const& _addr, bytes const& _oldCode):
-            kind(Code), address(_addr), oldCode(_oldCode)
+    Change(Address const& _addr, bytes const& _oldCode, u256 const& _oldVersion)
```

Comment:
I forgot what's the exact reason to save the old code, I'll try to remember, but if it's needed, probably we need to save the version, too, and restore during rollback.

Reply:
Apparently this is not needed anymore, I think I'll remove saving the old code in https://github.com/ethereum/aleth/pull/5641, then will rebase this.